api/group/repository/remove.js

```diff
@@ -0,0 +1,9 @@
+import {Group} from '../../common/UserAndGroupRepository.js';
+
+export default async function remove(GroupId) {
+  await Group.destroy({
```

Comment:
Add add handling of removing userGroup row after group- was removed
If any record gets removed from the DB, then all linked records should be removed from UserGroupas well.

Reply:
you could add { onDelete: 'cascade' } for models in your relations.js file